lib/msf/core/post/file.rb

```diff
@@ -578,6 +579,10 @@ def copy_file(src_file, dst_file)
 
 protected
 
+  def powershell_read_file(file_name)
```

Comment:
Can you rename this to `_read_file_powershell` so it's like `_read_file_meterpreter` and place it under that definition so they're together?

Reply:
sure :+1: